### PR TITLE
docs - troubleshooting visualizations

### PR DESCRIPTION
**Pull request by nllho**
New guide for common viz problems, plus some cleanup:

- Distinguish viz settings for a question vs. a dashboard card.
- Promote headings on the Troubleshooting index (for the TOC).
- Replace some footer links with far superior inline links ;)

### CODE REVIEW

File: docs/troubleshooting-guide/visualization.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Formatting dashboard cards
+
+1. Make sure that you're making and saving changes from the [card's settings](../dashboards/introduction.md#changing-a-cards-visualization-settings) (_not_ the original question's settings).
+2. [Reset your card's visualization settings](../dashboards/introduction.md#resetting-a-cards-visualization-settings).
+
+**Explanation**
+
+The visualization settings on a card are independent of the settings on the original question.
+
+When you first create a question, your selected visualization type is saved along with the query. When you add that question to a dashboard, the dashboard will display the same visualization as the original question by default. You can override the original visualization type by using the [card's visualization settings](../dashboards/introduction.md#changing-a-cards-visualization-settings).
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/visualization.md:22]:
```suggestion
The visualization settings on a card are independent of the settings on the original question. When you first create a question, your selected visualization type is saved along with the query. When you add that question to a dashboard, the dashboard will display the same visualization as the original question by default. You can override the original visualization type by using the [card's visualization settings](../dashboards/introduction.md#changing-a-cards-visualization-settings).
```

File: docs/troubleshooting-guide/visualization.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Formatting dates and times
+
+See [Troubleshooting dates and times](./timezones.md).
+
+## Uploading a file for a custom map
+
+If you're unable to add a [GeoJSON file for a custom map](../configuring-metabase/custom-maps.md#custom-region-maps), try reducing the size of the file to 5MB or less.
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/visualization.md:30]: For these short sections that only really serve to reroute people elsewhere: do we need them? My concern is that they could muddy search results, which could put additional clicks between a search and a relevant answer.

File: docs/troubleshooting-guide/visualization.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Go to your SQL question and [change the visualization type](../questions/sharing/visualizing-results.md) to a table. Then, check if any of the following situations apply to the raw query results:
+
+- [Aggregations (counts, sums, etc.) are wrong](https://www.metabase.com/learn/debugging-sql/sql-logic#aggregated-results-counts-sums-etc-are-wrong).
+- [Results have duplicated rows](https://www.metabase.com/learn/debugging-sql/sql-logic-duplicated-data).
+- [Results are missing rows](https://www.metabase.com/learn/debugging-sql/sql-logic-missing-data).
+
+**Explanation**
+
+If your question or dashboard card is powered by a handwritten [SQL query](../questions/native-editor/writing-sql.md) rather than the [query builder](../questions/query-builder/introduction.md), your visualization is going to be more sensitive to changes in the underlying data (for example, renamed fields, or the sudden appearance of a wild null value). To learn more, read about [Common reasons for unexpected query results](https://www.metabase.com/learn/debugging-sql/sql-logic#common-reasons-for-unexpected-query-results).
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/visualization.md:42]: WILD NULL

**Review by jeff-bruemmer — APPROVED** [review]
Thanks, @nllho!

File: docs/troubleshooting-guide/my-dashboard-is-slow.md [hunk 4]
```diff
@@ -117,6 +117,13 @@ Similarly, you probably don't need indexes for simple tables with a few tens of
 2. Check the performance logs for the server where Metabase is running.
 3. If either is a bottleneck, upgrade to a more powerful server or one with more memory.
 
+## Do you have a different problem?
```
[the remark this answers is not in the document]
Comment (nllho) [reply docs/troubleshooting-guide/my-dashboard-is-slow.md:120]: I changed it to "Related problems" instead. "Other problems" kind of felt like "Further reading" to me, like we're telling people to check out these other problems they might have 😆

**Pull request merged**